Composer/packages/client/src/pages/design/DebugPanel/TabExtensions/DiagnosticsTab/DiagnosticList.tsx

```diff
@@ -12,26 +12,19 @@ import {
 } from 'office-ui-fabric-react/lib/DetailsList';
 import { Sticky, StickyPositionType } from 'office-ui-fabric-react/lib/Sticky';
 import { TooltipHost } from 'office-ui-fabric-react/lib/Tooltip';
-import { ScrollablePane, ScrollbarVisibility } from 'office-ui-fabric-react/lib/ScrollablePane';
 import { FontIcon } from 'office-ui-fabric-react/lib/Icon';
-import { useMemo, useState } from 'react';
 import formatMessage from 'format-message';
-import { RouteComponentProps } from '@reach/router';
-import { useRecoilValue } from 'recoil';
 import { mergeStyleSets } from 'office-ui-fabric-react/lib/Styling';
 import { FontSizes } from '@uifabric/fluent-theme';
 import { css } from '@emotion/core';
 
-import { Pagination } from '../../components/Pagination';
-import { diagnosticsSelectorFamily } from '../../recoilModel/selectors/diagnosticsPageSelector';
-
-import { IDiagnosticInfo } from './types';
+import { IDiagnosticInfo } from '../../../../diagnostics/types';
 
 // -------------------- Styles -------------------- //
 
 const icons = {
-  Error: { iconName: 'ErrorBadge', color: '#A80000', background: '#FED9CC' },
-  Warning: { iconName: 'Warning', color: '#8A8780', background: '#FFF4CE' },
+  Error: { iconName: 'StatusErrorFull', color: '#EB3941' },
```

Comment:
Can we replace hardcoded color variables with SharedColors, CommunicationColors?

Reply:
Seems we are not using standard colors provided by fluent ui... I can only map the #DFDFDF to tint50 by searching the source code but unfortunately it's not exported.